libs/providers/rhv.py

```
@@ -309,6 +330,65 @@ def get_template_by_name(self, name: str) -> Any:
+        if not mappings:
+            raise ValueError(
+                f"No networks found for templates {template_names}. "
+                f"Ensure templates have network interfaces configured with valid vNIC profiles."
+            )
+
+        return mappings
+
+    def get_vm_or_template_networks(
+        self,
+        names: list[str],
+        inventory: ForkliftInventory,
```

Ruff (0.14.13)

Unused method argument: `inventory`

(ARG002)